src/adm/filter.rs

```diff
@@ -107,6 +119,90 @@ impl AdmFilter {
         l_sentry::report(&merged_tags, sentry::event_from_error(error));
     }
 
+    /// check to see if the bucket has been modified since the last time we updated.
+    pub async fn requires_update(&self) -> HandlerResult<bool> {
+        // don't update non-bucket versions (for now)
+        if !self.is_cloud() {
+            return Ok(false);
+        }
+        if let Some(bucket) = &self.source_url {
+            let host = bucket
+                .host()
+                .ok_or_else(|| {
+                    HandlerError::internal(&format!("Missing bucket Host {:?}", self.source))
+                })?
+                .to_string();
+
+            let obj = cloud_storage::Object::read(&host, bucket.path())
```

Comment:
Testing this w/ e.g. `CONTILE_ADM_SETTINGS=gs://pjenvey-test/adm_settings.json` --

this `read` and the `Object::download` call in settings both fail w/ 

```
GoogleError { domain: "global", reason: NotFound, message: "No such object: pjenvey-test//adm_settings.json"`
```

but they work with the leading slash removed from `bucket.path()`

Reply:
Huh, that worked for me. Will investigate.